### ISSUE TITLE
pip install fails with "no module dateutil" in fresh virtual environment

### ISSUE THREAD

**Issue by jvmncs** [minimaxir/simpleaichat#1]
```console
$ conda create -n simple-ai python && conda activate simple-ai
...
$ pip install simpleaichat
...
$ python
Python 3.11.3 (main, Apr 19 2023, 23:54:32) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import simpleaichat
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/.../site-packages/simpleaichat/__init__.py", line 1, in <module>
    from .simpleaichat import AIChat, AsyncAIChat
  File "/.../site-packages/simpleaichat/simpleaichat.py", line 3, in <module>
    import dateutil
ModuleNotFoundError: No module named 'dateutil'
```

**Comment by jvmncs** [comment]
`pip install python-dateutil` fixed it for me

**Comment by minimaxir** [comment]
Yep, looks like a missed a dependency (it's common in a lot of packages). Will add to `setup.py`.

**Comment by siAyush** [comment]
Hi @minimaxir 
Can i take this up?

**Comment by minimaxir** [comment]
fixed and pushed v`0.1.1` which has the dependency.
